masonry/src/widgets/sized_box.rs

```diff
@@ -327,7 +327,9 @@ impl SizedBox {
 
 // --- MARK: WIDGETMUT ---
 impl SizedBox {
-    #[expect(missing_docs, reason = "TODO")]
+    /// Give this container a child widget.
+    ///
+    /// If this container already has a child, it will be overwritten.
     pub fn set_child(this: &mut WidgetMut<'_, Self>, child: impl Widget) {
```

Comment:
Incidentally, this was an extremely surprising API to me. I don't know that it causes any complication, but it didn't match my mental model of how widgets like this should be handled.